### PR TITLE
PR-596 => develop

### PR DESCRIPTION
**Pull request by nmsger-isento**
- [ ] All acceptance criteria have been implemented.

- [ ] Tester feedback has been incorporated.

- [ ] If necessary, a documentation has been created/ code-comments written.

- [ ] At least one successful code review has been conducted.

- [ ] If necessary, an appointment for knowledge transfer has been arranged.

- [ ] Dependencies on other repositories have been checked and implemented if necessary.

- [ ] At least one successful setup-test on a virtual machine was done.

- [ ] At least one successful setup-test on a raspberry pi/ physical pib was done.

### CODE REVIEW

File: ros_packages/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        libusb-1.0-0-dev \
+        flac \
+        libgl1 \
+        zsh \
+        git \
+        software-properties-common \
+        wget
+
+# OAK-D-LITE CAMERA SETUP (TAKEN FROM: https://github.com/luxonis/depthai-ros/blob/humble/Dockerfile)
+# RUN sh -c "$(wget https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh -O -)"
```
Comment (niklasIsento) [inline ros_packages/Dockerfile:28]: Remove comments

File: ros_packages/motors/motors/motor_control.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
 from pib_api_client import motor_client
 from pib_motors.motor import name_to_motors, motors
 from pib_motors.bricklet import ipcon      
-import sys
-sys.path.append('/home/pib/ros_working_dir/src/motors/utils')
-import update_bricklet_uids
-
+# import sys
+# sys.path.append('/home/pib/ros_working_dir/src/motors/utils')
```
Comment (niklasIsento) [inline ros_packages/motors/motors/motor_control.py:10]: Remove comments

File: ros_packages/motors/motors/motor_current.py [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
 from tinkerforge.bricklet_servo_v2 import BrickletServoV2
 import rclpy
 from rclpy.node import Node
 from diagnostic_msgs.msg import DiagnosticStatus
 from diagnostic_msgs.msg import KeyValue
-import sys
-sys.path.append('/home/pib/ros_working_dir/src/motors/utils')
-import update_bricklet_uids
+from pib_motors.update_bricklet_uids import *
+# import sys
```
Comment (niklasIsento) [inline ros_packages/motors/motors/motor_current.py:10]: Remove

File: ros_packages/voice_assistant/voice_assistant/assistant.py [hunk 4]
```diff
@@ -25,8 +25,7 @@
 RECEIVE_CHAT_MESSAGE_WAITING_PERIOD_SECONDS = 0.1
 MAIN_LOOP_RECEIVE_SIGNAL_WAITING_PERIOD_SECONDS = 0.05
 
-
-VOICE_ASSISTANT_PATH_PREFIX = "/home/pib/ros_working_dir/src/voice_assistant"
+VOICE_ASSISTANT_PATH_PREFIX = os.getenv("VOICE_ASSISTANT_DIR", "/home/pib/ros_working_dir/src/voice-assistant")
```
Comment (niklasIsento) [inline ros_packages/voice_assistant/voice_assistant/assistant.py:28]: Please change /voice-assistant to /voice_assistant

File: docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      - ./pib_api/flask/:/app
+    environment:
+      - SQLALCHEMY_DATABASE_URI=sqlite:////app/pibdata.db
+    command: "bash -c 'flask --app run db upgrade && flask --app run seed_db && flask --app run run --host 0.0.0.0 --debug'"
+
+  ros-packages:
+    image: ros_packages
+    privileged: true
+    build:
+      context: .
```
[the remark this answers is not in the document]
Comment (nmsger-isento) [reply docker-compose.yaml:27]: If not specified, context defaults to `.`, so in that sense it is not necessary to write down, but for the sake of clarity it's best to define it nevertheless.

**Review by niklasIsento — APPROVED** [review]

**Pull request merged**